main.js

```diff
@@ -1,9 +1,30 @@
-for(let i = 1; i < 100; i++){
-  if (i % 15 === 0){
-    console.log(`FizzBuzz ${i}`);
-   } else if (i % 5 === 0){
-    console.log(`Buzz ${i}`);
-   } else if (i % 3 === 0){
-    console.log(`Fizz ${i}`);
- }
-}
\ No newline at end of file
+"use strict";
+console.log("=========================");
+console.log("現在持っているタスク一覧");
+console.log("=========================");
+const list = [
+  {task:"机を片付ける",genre:"掃除"},
+  {task:"牛乳を買う",genre:"買い物"},
+  {task:"散歩する",genre:"運動"},
+];
+  for (let i = 0; i < list.length; i++ ){
+    console.log(`${i} : [内容] ${list[i].task}、[ジャンル] ${list[i].genre}`);
+}
```

Comment:
`forEach` を使う方がすっきり書けますよ。

```suggestion
list.forEach((task, index) => {
  console.log(`${i} : [内容] ${task.task}、[ジャンル] ${task.genre}`);
});
```

Reply:
${i}ですが、
Uncaught ReferenceError: i is not defined
と指摘うけました。
indexで定義されているので、${index}になるのではないんでしょうか？
ご教示お願いいたします。

Reply:
そうですね、おっしゃる通りです。